plugins/env/app/controllers/admin/environment_variable_groups_controller.rb

```diff
@@ -34,10 +34,10 @@ def destroy
     def preview
       @groups = if params[:group_id]
         @group = EnvironmentVariableGroup.find(params[:group_id])
-        SamsonEnv.env_groups(Stage.new(environment_variable_groups: [@group], deploy_groups: DeployGroup.all))
+        SamsonEnv.env_groups(Stage.new(project: Project.new(environment_variable_groups: [@group]), deploy_groups: DeployGroup.all))
```

Comment:
I guess this applies to other methods around this plugin as well so no prob if you want to leave it as is